docs/timezone/timezone.md

```diff
@@ -13,7 +14,10 @@ For legacy or unsupported environments, please use a proper [polyfill](https://g
 dayjs.extend(utc)
 dayjs.extend(timezone)
 
-dayjs.tz("2014-06-01 12:00", "America/New_York")
+// current time zone is 'Europe/Berlin' (offset +01:00)
+// Parsing
+dayjs.tz("2013-11-18T11:55:20", "America/Toronto") // '2013-11-18T11:55:20-05:00'
```

Comment:
will actually dayjs is able to parse string like `2014-06-01 12:00`

Reply:
Of course, like always you are right.

I wanted to show the results of the different ways of parsing. Changing from `2014-06-01 12:00` to `2013-11-18T11:55:20` was just because the in the issue that caused this pr here an ISO 8601 date was used and in the [documentation](https://day.js.org/docs/en/parse/string) we also only talk about ISO dates (and that requires a 'T' between the date and the time part).

Should I change to a date with a space instead of a 'T'?

Reply:
I see. 

If so, could we use "a date with a space" here in the timezone doc, and add another date string other than ISO in the "parse string" doc to make that clear?

Reply:
done